packages/test/src/admin-utils.ts

```diff
@@ -166,20 +191,27 @@ export async function assignProductToCollection(
   };
 }
 
-export async function createCollectionContainingProduct(
+export async function createCollectionContainingVariants(
   adminClient: SimpleGraphQLClient,
   translationFields: Omit<
     CreateCollectionTranslationInput,
     'customFields' | 'languageCode'
   >,
-  productId: string,
+  variants: ProductVariant[],
```

Comment:
I see you only use the `variant.id` of each variant. A good practice is to only accept what you need as parameter for your function. So, in this case it would be better to accept a list of variant ids `variantIds: ID[]`